parsers/UY.py

```diff
@@ -1,163 +1,224 @@
 #!/usr/bin/python3
 
-import re
-from datetime import datetime
+import io
+from datetime import datetime, timedelta
 from logging import Logger, getLogger
 from typing import Optional
 
 import arrow
-import dateutil
+import pandas as pd
+import pytz
 
 # BeautifulSoup is used to parse HTML to get information
 from bs4 import BeautifulSoup
 from requests import Session
 
-tz = "America/Montevideo"
+from electricitymap.contrib.config.constants import PRODUCTION_MODES
```

Comment:
I think you can just do config.constants the full path does not work on windows if I'm not mistaken

Reply:
Actually this one will work cause config is the only folder in there and wont work with config.constant as it's the only one that is not a symlink. (and config.constants would point to the other config folder and it don't have constants file).

It's all a little confusing 😅